proto/service_ticket.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+// ServiceTicketApi is a service which provides an interface for creating and updating service tickets in a
+// third party system. e.g. a CAFM.
+service ServiceTicketApi {
+  rpc CreateTicket(CreateTicketRequest) returns (Ticket) {}
+  rpc UpdateTicket(UpdateTicketRequest) returns (Ticket) {}
+  // there are intentionally no retrieve and delete operations. smart core is not designed to be a CAFM.
+  // we need to be able to raise tickets in the external system and possibly update them if we discover
+  // more useful info to help resolve the issue. We do not want to be managing the tickets in smart core.
+}
+
+// Ticket represents a service ticket in a third party system.
+message Ticket {
+  // Id is blank when creating a ticket, and is filled in by the external system. The ID is then used to update the ticket.
+  string id = 1;
+  // Short description of the issue being reported.
+  string description = 2;
+  // Full details on the issue being reported. This should include all the available information to help resolve the issue.
+  string details = 3;
```

Comment:
Can we adjust these slightly. I think `summary` and `description` (instead of `description` and `details`) would align better with other uses of description in our apis.